Makefile

```diff
@@ -1,39 +1,82 @@
-.PHONY: build
+# Set the default target to build all
+.DEFAULT_GOAL := all
+
+# Declare all targets as phony targets
+.PHONY: all build swagger swagger-run job core clean lint ut help
+
+# Build all targets
+all: tidy gen format lint cover build
+
+## build: Build the project for horizon
 build:
-	mkdir -p bin && export CGO_ENABLED=0 GOOS=linux && go build -o bin/app -ldflags '-s -w' ./core/main.go
+	mkdir -p bin && export CGO_ENABLED=0 && go build -o bin/app -ldflags '-s -w' ./core/main.go
 
-.PHONY: swagger
+## format: Gofmt (reformat) package sources (exclude vendor dir if existed).
+.PHONY: format
+format: tools.verify.golines tools.verify.goimports
```

Comment:
goimport include gofmt,so we use just goimport  is ok.
`# go source files, ignore vendor directory
SRC = $(shell find . -type f -name '*.go' -not -path "./vendor/*")

imports:
	@goimports -l -w $(SRC)
`